sdcm/commit_log_check_thread.py

```diff
@@ -0,0 +1,192 @@
+import traceback
+import logging
+import time
+import re
+import threading
+from dataclasses import dataclass
+from distutils.util import strtobool
+from sdcm.sct_events.database import CommitLogCheckErrorEvent
+from sdcm.rest.remote_curl_client import RemoteCurlClient
+
+
+@dataclass
+class CommitlogConfigParams:
+    def __init__(self, db_cluster, ):
+        logger = logging.getLogger(self.__class__.__name__)
+        with db_cluster.cql_connection_patient(
+                node=db_cluster.nodes[0],
+                connect_timeout=300,) as session:
+            self.total_space = int(session.execute(
+                "SELECT value FROM system.config WHERE name='commitlog_total_space_in_mb'").one().value)
```

Comment:
If we continue with the approach with the API, this is probably not necessary anymore

Reply:
fixed